src/net/ossl.cc

```diff
@@ -437,6 +440,23 @@ class tls::certificate_credentials::impl {
         }
     }
 
+    void enable_crl_checking() {
```

Comment:
awesome maybe we can add a simple crl unit test to prevent regressions

Reply:
There's an issue already for this: https://redpandadata.atlassian.net/browse/CORE-2078